test/functional/test_output_buffer.py

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+#
+# Tests for libxlsxwriter.
+#
+# Copyright 2014-2022, John McNamara, jmcnamara@cpan.org
+#
+
+import base_test_class
+
+class TestCompareXLSXFiles(base_test_class.XLSXBaseTest):
+    """
+    Test file created with libxlsxwriter against a file created by Excel.
+
+    """
+
+    def test_output_buffer01(self):
+        self.run_exe_test('test_output_buffer01', 'simple01.xlsx')
```

Comment:
Extra points for working this out. I can never remember the syntax for running a test case against a differently named target file, and I wrote the test code. ;-)